README.md

```diff
@@ -58,7 +115,7 @@ class ACLineSegmentCustom(ACLineSegment):
         return "ACLineSegment"
 ```
 
-### Namespace
+#### Namespace
 
 In the serialisation, the namespace of all attributes is `cim` (`"http://iec.ch/TC57/2013/CIM-schema-cim16#"`) by default.
 The serialisation is not done by PyCGMES (yet), but if you want a custom namespace for an attribute,
```

Comment:
This works for namespaces of classes, but 1) it implies that if you define a new profile, you have to specify namespace for every attribute you attach from that namespace; 2) I am not sure how it would work to define the namespace for a custom class that belongs to a custom profile.

Reply:
1) this is currently correct. 
2) classes do not have namespaces, only attributes if I understood correctly?

If I understand it correctly, you would like to have a way to set a "class namespace", which would then apply to all its attribute? That feels like a good idea.

Reply:
following what's defined in: https://www.entsoe.eu/Documents/CIM_documents/Grid_Model_CIM/CGMES_Profiling_User_Guide_v1.0.pdf

when you create an extension of CIM (profiles, classes, attributes), you usually define a namespace for it.

this is actually very convenient naming wise, because you could define a custom ACLineSegment as `my:ACLineSegment` instead of `cim:CustomACLineSegment`

Reply:
I see. I had missed this indeed. It should be an easy fix.